File: src/Commands/others/sus.ts
import { Command } from "../../Interfaces";

export const command: Command = {
  name: "sus",
  description: "Sends sus dog",
  category: "Entertainment",
  aliases: [],
  run: async (variables) => {
    variables.client.messager.send_files(variables, "SUS!", ["./images/imposter_dog.jpg"]);
  }
};

File: src/bot/commands/entertainment/sus.rs
use crate::bot::commands::{Context, Error};

const IMPOSTER_DOG: &[u8] = include_bytes!(concat!(
    env!("CARGO_MANIFEST_DIR"),
    "/images/imposter_dog.jpg"
));

/// Sends sus dog
#[poise::command(slash_command, prefix_command, category = "Entertainment")]
pub async fn sus(ctx: Context<'_>) -> Result<(), Error> {
    message!(
        file,
        bytes,
        ctx,
        "IT'S SUS!",
        IMPOSTER_DOG;
        "imposter_dog.jpg",
        false
    );

    Ok(())
}
